src/test/codewhisperer/explorer/codewhispererNode.test.ts

```diff
@@ -78,5 +78,27 @@ describe('codewhispererNode', function () {
             assert.ok(ssoSignInNode)
             assert.ok(learnMorenNode)
         })
+
+        it('should get correct child nodes if user is  connected', function () {
+            sinon.stub(AuthUtil.instance, 'isUsingSavedConnection').get(() => true)
+            isConnectionValid.returns(true)
+            isConnected.returns(true)
+            const node = codewhispererNode
+            const children = node.getChildren()
+
+            const pauseAutoSuggestionsNode = children.find(
+                c => c.resource.id == 'aws.codeWhisperer.toggleCodeSuggestion'
+            )
+            const getingStartedNode = children.find(c => c.resource.id == 'aws.codeWhisperer.gettingStarted')
+            const openCodeReferenceNode = children.find(c => c.resource.id == 'aws.codeWhisperer.openReferencePanel')
+            const runSecurityScanNode = children.find(c => c.resource.id == 'aws.codeWhisperer.security.scan')
+
+            assert.strictEqual(children.length, 4)
+
+            assert.ok(pauseAutoSuggestionsNode)
+            assert.ok(runSecurityScanNode)
+            assert.ok(openCodeReferenceNode)
+            assert.ok(getingStartedNode)
```

Comment:
would it be less verbose and more readable to do something like:

```
assert.deepStrictEqual([...], children.map(o => o.resource.id)
```

Reply:
Updated !